Makefile

```diff
@@ -182,6 +203,14 @@ devenv-down: ## Stop optional services.
 	test -f docker-compose.yaml && \
 	docker-compose down || exit 0;
 
+devenv-postgres:
+	@cd devenv; \
+	sources=postgres_tests
```

Comment:
Not sure if we need this, given that we can use:
`make devenv sources=postgres_tests`